composites/SnippetPreview/components/Icon.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+/**
+ * Returns the Icon component.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Icon component.
+ */
+export const Icon = ( props ) => {
+	const IconComponent = styled( props.icon )`
+		float: left;
+		width: ${ props => props.iconSize }px;
+		height: ${ props => props.iconSize }px;
+		fill: ${ props => props.iconColor };
+	`;
+
+	return <IconComponent aria-hidden="true" iconSize={ props.iconSize } iconColor={ props.iconColor } />;
```

Comment:
I'm getting `Warning: Unknown props `iconSize`, `iconColor` on <svg> tag. `
because those props are applied directly to the svg element and interpreted as non-standard attributes. `IconComponent` is actually a SVG element, we should probably use attributes width, height, fill, and pass them the values.